invertedindex/ds_index.go

```diff
@@ -137,7 +148,7 @@ func (d *DataStoreIndex) GetShardsForCid(c cid.Cid) ([]shard.Key, error) {
 	return shardKeys, nil
 }
 
-func (d *DataStoreIndex) NCids() (uint64, error) {
+func (d *DataStoreIndex) NMultihashes() (uint64, error) {
```

Comment:
Nit, but maybe calling this just `Length`? It is clear that this index contains `mh -> shards`, so `Length` sounds like it would return the number of items in the index.

```suggestion
func (d *DataStoreIndex) NMultihashes() (uint64, error) {
```